src/sdk.js

```diff
@@ -577,62 +142,147 @@ const authWithIdpInterceptors = [
   new AddAuthTokenResponse(),
 ];
 
+const formatError = e => {
+  /* eslint-disable no-param-reassign */
+  if (e.response) {
+    const { status, statusText, data } = e.response;
+    Object.assign(e, { status, statusText, data });
+    delete e.response;
+  }
+
+  if (e.ctx) {
+    // Remove context `ctx` from the error response.
+    //
+    // `ctx` is SDK internal and should be exposed as a part of the
```

Comment:
```suggestion
    // `ctx` is SDK internal and shouldn't be exposed as a part of the
```

We don't want to expose ctx in SDK public API, right?